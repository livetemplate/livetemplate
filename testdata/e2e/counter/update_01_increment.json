{
  "0": "Simple Counter",
  "1": "5",
  "2": "positive",
  "3": "<p>Counter is positive</p>",
  "4": "2023-01-01 10:05:00",
  "5": "counter-12345",
  "s": [
    "<h1>",
    "</h1><div><p>Counter:",
    "</p><p>Status:",
    "</p>",
    "</div><footer><p>Last updated:",
    "</p><p>Session:",
    "</p></footer>"
  ]
}
