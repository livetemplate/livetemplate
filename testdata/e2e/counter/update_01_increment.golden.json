{
  "0": "Simple Counter",
  "1": "5",
  "2": "positive",
  "3": "\u003cp\u003eCounter is positive\u003c/p\u003e",
  "4": "2023-01-01 10:05:00",
  "5": "counter-12345",
  "s": [
    "\u003ch1\u003e",
    "\u003c/h1\u003e\u003cdiv\u003e\u003cp\u003eCounter: ",
    "\u003c/p\u003e\u003cp\u003eStatus: ",
    "\u003c/p\u003e ",
    " \u003c/div\u003e\u003cfooter\u003e\u003cp\u003eLast updated: ",
    "\u003c/p\u003e\u003cp\u003eSession: ",
    "\u003c/p\u003e\u003c/footer\u003e"
  ]
}